javascript-practice/src/js/views/userView.js

```diff
@@ -17,7 +17,7 @@ export default class UserView {
     this.fullNameEl = document.getElementById('fullName');
     this.headerNameEl = document.getElementById('headerName');
     this.loginFormEl = document.getElementById('login-form');
-    this.logoutEl = document.getElementById('logout');
+    this.logoutEl = document.getElementById('list-drop');
```

Comment:
logoutEl and list-drop doesn't not match meaning. Btw ID should format like myCustomId, not my-custom-id